src/components/config-tab.js

```diff
@@ -1,34 +1,32 @@
 'use strict';
 
-const _ = require('lodash');
 const React = require('react');
+const { createContainer } = require('sovereign');
 
 const Checkbox = require('../primed/checkbox');
+const configStore = require('../stores/configStore');
+const { setConfig } = require('../actions/configActions');
 
 class ConfigTab extends React.Component {
   constructor(...args){
     super(...args);
-    this.state = { autoFill: false };
-    this.toggle = this.toggle.bind(this);
   }
-  componentDidMount(){
-    chrome.storage.sync.get('autoFill', (res)=>{
-      this.setState({ autoFill: res.autoFill });
-    });
+  config(event){
+    setConfig(event.target.checked);
```

Comment:
this is a good example of what I was talking about in my other comments about event objects being the payload to an action.  I like this pattern a lot more than passing the event object to a store